realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -404,6 +404,192 @@ public boolean isValid() {
         return this;
     }
 
+    // In
+
+    /**
+     * In comparison. This allows you to test if objects match any value in an array of values.
+     *
+     * @param fieldName the field to compare.
+     * @param values values to compare with and it cannot be null or empty.
+     * @return the query object.
+     * @throws java.lang.IllegalArgumentException if values is empty or do not match class or field type.
+     */
+    public RealmQuery<E> in(String fieldName, String... values) {
```

Comment:
I am not a big fan of this API, it is very hard to decipher what is the field name and what is the parameter. 
```
in("foo", "bar", "baz") // hard to decipher 
in("foo", new String[] { "bar", baz" }), much easier to read, but a bit harder to type
```
IMO readability should always win.
Also moving the `Case` to the middle is not in line with the rest of the API.

IMO we should use `in(String fieldName, String[] values, Case case)`  instead of var args.

Thoughts @realm/java ?